frontend/model/contracts/chatroom.js

```diff
@@ -129,26 +130,7 @@ sbp('chelonia/defineContract', {
       }
     }
   },
-  getters: {
-    currentChatRoomState (state) {
-      return state
-    },
```

Comment:
This is being removed and not re-added anywhere. That's a problem because it is still referenced elsewhere in the project.

Reply:
It's not referenced elsewhere